### PR TITLE
Making it a bit easier to start with weaviate

### PR DESCRIPTION
**Pull request by raam86**
As a new developer I will scan the readme for these type of instructions. Failing to find these may cause me to come back later or even never try out this awesome tech!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,15 @@
     * Deploy and maintain your ML models in production reliably and efficiently.
     * Weaviate's modular design allows you to easily package any custom trained model you want.
     * Smooth and accelerated handover of your Machine Learning models to engineers.
+## Install
```
Comment (raam86) [inline README.md:33]: Install section is an essential part of every readme; some examples: 
https://github.com/apache/airflow#installing-from-pypi
https://github.com/dbt-labs/dbt-core#getting-started
https://github.com/prestodb/presto#running-the-cli

**Review by raam86 — COMMENTED** [review]
let me know what you think

**Review by etiennedi — APPROVED** [review]

**Comment by etiennedi** [comment]
Great idea, LGTM.

**Review by aliszka — APPROVED** [review]

**Comment by etiennedi** [comment]
@bobvanluijt With the recent changes for "Getting started" does it still make sense to have a link in the readme that uses a full module-based demo dataset or should we rather go for something without modules first?

**Pull request closed without merging**